frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -59,6 +60,80 @@ export const validationSchema = yup.object().shape({
       .string()
       .matches(pathRegex, { message: 'Path must start with /.', excludeEmptyString: true }),
   }),
+  limits: yup.object().shape({
+    cpu: yup.object().shape({
+      request: yup
+        .number()
+        .integer('Request must be an Integer.')
+        .min(0, 'Request must be greater than or equal to 0.')
+        .test({
+          test(request) {
+            const { requestUnit } = this.parent;
+            const { limit } = this.parent;
+            const { limitUnit } = this.parent;
+            return (
+              convertToBaseValue(`${request}${requestUnit}`) <=
+              convertToBaseValue(`${limit}${limitUnit}`)
+            );
+          },
+          message: 'CPU request must be less than or equal to limit.',
+        }),
+      requestUnit: yup.string('Unit must be millicores or cores.'),
+      limitUnit: yup.string('Unit must be millicores or cores.'),
+      limit: yup
+        .number()
+        .integer('Limit must be an Integer.')
+        .min(0, 'Limit must be greater than or equal to 0.')
+        .test({
+          test(limit) {
+            const { request } = this.parent;
+            const { requestUnit } = this.parent;
+            const { limitUnit } = this.parent;
+            return (
+              convertToBaseValue(`${limit}${limitUnit}`) >=
+              convertToBaseValue(`${request}${requestUnit}`)
+            );
+          },
+          message: 'CPU limit must be greater than or equal to request.',
+        }),
+    }),
+    memory: yup.object().shape({
+      request: yup
+        .number()
+        .integer('Request must be an Integer.')
+        .min(0, 'Request must be greater than or equal to 0.')
+        .test({
+          test(request) {
+            const { requestUnit } = this.parent;
+            const { limit } = this.parent;
+            const { limitUnit } = this.parent;
+            return (
+              convertToBaseValue(`${request}${requestUnit}`) <=
+              convertToBaseValue(`${limit}${limitUnit}`)
+            );
+          },
+          message: 'Memory request must be less than or equal to limit.',
+        }),
+      requestUnit: yup.string('Unit must be Mib or Gib or Mb or GB.'),
+      limit: yup
+        .number()
+        .integer('Limit must be an Integer.')
+        .min(0, 'Limit must be greater than or equal to 0.')
+        .test({
+          test(limit) {
+            const { request } = this.parent;
+            const { requestUnit } = this.parent;
+            const { limitUnit } = this.parent;
+            return (
+              convertToBaseValue(`${request}${requestUnit}`) <=
+              convertToBaseValue(`${limit}${limitUnit}`)
+            );
+          },
+          message: 'Memory limit must be greater than or equal to request.',
+        }),
+      limitUnit: yup.string('Unit must be Mib or Gib or Mb or GB.'),
```

Comment:
Seeing this in several messages.

```suggestion
      limitUnit: yup.string('Unit must be MiB or GiB or MB or GB.'),
```

But you probably want to match exactly what's in the dropdown, so I'd suggest

```suggestion
      limitUnit: yup.string('Unit must be Mi or Gi or M or G.'),
```